examples/index.html

```diff
@@ -1,10 +1,10 @@
 <!doctype html>
-<iframe width="600px" height="800px" src="http://gohone.com/Quiz/5407769cfae50de35b93a1c2?debug=true&embed=true" data-hone frameborder="0" data-resize="true">You Dont Support Iframes</iframe>
-<script src="../build/embed.js" defer></script>
+<iframe width="600px" height="800px" src="https://gohone.com/Quiz/5407769cfae50de35b93a1c2?debug=true&embed=true" data-hone frameborder="0" data-resize="true">You Dont Support Iframes</iframe>
```

Comment:
Yeah I do but that is just I assume because of the fonts they use for the quiz (not https) everything still logs correctly.